platform/darwin/src/MGLStyle.mm

```diff
@@ -254,5 +257,21 @@ - (void)removeStyleClass:(NSString *)styleClass
     }
 }
 
+- (void)setImage:(MGLImage *)image forName:(NSString *)name
+{
+    if (image && name)
```

Comment:
Just pushed it. I had to add a second argument with description otherwise I got error `too few arguments provided to function-like macro invocation`.